app/templates/agenda.html

```diff
@@ -60,4 +69,5 @@ <h1 id="dates">{{ start_date.strftime("%d/%m/%Y") }} - {{ end_date.strftime("%d/
     {% endfor %}
   </div>
 
+</div>
```

Comment:
You sure you need to close this div here?

Reply:
done